src/main.dev.ts

```diff
@@ -289,22 +291,27 @@ app.on('ready', async () => {
         menu.tray.setContextMenu(trayMenuHelper.createContextMenu());
       });
     });
-    ipcMain.on(LISTENER_LOG, (evt, id: string) => {
+    ipcMain.on(LISTENER_LOG, (evt, args) => {
       const sendTo = evt?.sender ? evt.sender : mainWindow?.webContents;
-      const stream = listenerClient.statusUpdates({
-        connectionId: id,
+      updateStream?.cancel();
+      updateStream = listenerClient.statusUpdates({
+        connectionId: args.id as string,
       } as StatusUpdatesRequest);
-      stream.on('data', (response) => {
+      updateStream.on('data', (response) => {
         sendTo?.send(LISTENER_LOG, {
-          msg: response.message,
+          msg: response as ConnectionStatusUpdate,
+          remoteAddr: args.remoteAddr,
         });
       });
+      // empty function otherwise causes fatal error on cancel !!!
+      updateStream.on('error', () => {});
```

Comment:
`console.error` maybe?

Reply:
no this is a hack to fix a library bug. we dont want the console to show anything